protocol.html

```diff
@@ -723,38 +723,124 @@ <h2 property="schema:name">Notification Data Model</h2>
             <div datatype="rdf:HTML" property="schema:description">
               <p>The content of a Solid Notification is defined in terms of a <cite><a href="https://www.w3.org/TR/activitystreams-core/">Activity Streams 2.0</a></cite> [<cite><a class="bibref" href="#bib-activitystreams-core">ACTIVITYSTREAMS-CORE</a></cite>]. As such, JSON-LD 1.0 is a required baseline serialization format, though other formats are not prohibited [<cite><a class="bibref" href="#bib-json-ld">JSON-LD</a></cite>]. JSON-LD version 1.1 is recommended when using JSON-LD serializations [<cite><a class="bibref" href="#bib-json-ld">JSON-LD11</a></cite>].</p>
 
-              <pre about="#notification-data-model-activity" property="schema:description" typeof="fabio:Script"><code>{</code>
+              <div class="note" id="solid-notification-json-ld-context" inlist="" rel="schema:hasPart" resource="#solid-notification-json-ld-context">
+                <h3 property="schema:name"><span>Note</span>: Solid Notification JSON-LD Context</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>The Solid JSON-LD context resource does not currently exist. An implementation may choose to augment the context definition.</p>
+                </div>
+              </div>
+
+              <p>Each time an event happens on a storage server, one or more notifications will be sent to the subscribing client in the following formats.</p>
+
+              <p>A notification activity MUST include a <a href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-type">type</a> field.</p>
+
+              <p>A notification activity SHOULD include an <a href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-id">id</a> field.</p>
+
+              <p>A notification activity MUST include an <a href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-actor">actor</a> field mapped to the URI of the authenticated agent that created the subscription. If an authenticated agent did not create the subscription, the actor field is not required.</p>
```

Comment:
There are privacy considerations here. _Requiring_ this field is, in my opinion, too strong. I would prefer to see this at a `MAY` or `SHOULD`, but definitely not `MUST`.